swagger-ui/dist/index.html

```diff
@@ -0,0 +1,60 @@
+<!-- HTML for static distribution bundle build -->
```

Comment:
- Move the top-level swagger-ui folder to 'js-apps/meep-swagger-ui'
- Remove dist folder if not required...

Reply:
done